cmd/src/actions_exec.go

```diff
@@ -184,13 +185,19 @@ Format of the action JSON files:
 			return err
 		}
 
-		err = validateActionDefinition(actionFile)
+		// Convert action file to JSON.
+		jsonActionFile, err := yaml.YAMLToJSONStrict(actionFile)
```

Comment:
Does this break with JSON files? Because that would be a blocker. We have to be backwards compatible.

Reply:
I think every JSON file should also be a valid YAML file. But if we support C-style `//` line comments in JSON, that would break.

Reply:
Correct, I figured that would be fine, but am open to discuss